include/json/features.h

```diff
@@ -52,6 +52,9 @@ class JSON_API Features {
 
   /// \c true if numeric object key are allowed. Default: \c false.
   bool allowNumericKeys_;
+
+  /// \c true if duplicate keys within an object should be rejected. Default: \c false.
+  bool rejectDupKeys_;
```

Comment:
Ok, thanks for the explanation.

I need both rejectDupKeys and error offsets, so neither class currently satisfies my needs. Do you expect any upcoming release to be allowed to break binary compatibility? I'm happy to expose structured error info through the CharReader interface (e.g. perhaps as a Json::Value), but don't see how to do that without breaking binary compatibility either.